bugsnag-android-unity/build.gradle

```diff
@@ -0,0 +1,25 @@
+buildscript {
+    repositories {
+        jcenter()
+        mavenCentral()
```

Comment:
Can get rid of mavenCentral as jcenter supplies the same dependencies and is faster